### ISSUE TITLE
Real Time with Bill Maher Overtime has episode search term messed up

### ISSUE THREAD

**Issue by adamwinn** [Sonarr/Sonarr#2976]
**Describe the bug**
Searching for a Real Time with Bill Maher Overtime episode, which was `S17E04 - Overtime`, Sonarr searched for `S00E1704`.  The season and episode numbers are in the wrong order.  It should be `S17E04` not `S00E1704`

**Logs**
```
19-3-3 01:58:32.7|Info|NzbSearchService|Searching 2 indexers for [Real Time with Bill Maher : 2019-02-08]
19-3-3 01:58:32.7|Debug|HDBits|Downloading Feed https://hdbits.org/api/torrents
19-3-3 01:58:32.7|Debug|BroadcastheNet|Downloading Feed http://api.broadcasthe.net/: getTorrents("(removed)", {"Category":"Episode","Name":"2019.02.08","Tvdb":"72231"}, 100, 0)
19-3-3 01:58:33.1|Debug|BroadcastheNet|Downloading Feed http://api.broadcasthe.net/: getTorrents("(removed)", {"Category":"Episode","Name":"S00E1704","Tvdb":"72231"}, 100, 0)
19-3-3 01:58:34.9|Debug|NzbSearchService|Total of 0 reports were found for [Real Time with Bill Maher : 2019-02-08] from 2 indexers
```
 - Sonarr Version: 2.0.0.5303
 - Operating System: Synology docker container
 - .net Framework (Windows) or mono (macOS/Linux) Version: [e.g. 4.5 or 5.12]

**Comment by Taloth** [comment]
Specials in scene often have S00Exx as format, which is why Sonarr searches both by the airdate and episode number (S00E1704) as given on [TheTVDB](https://www.thetvdb.com/series/real-time-with-bill-maher/seasons/0). You can't expect Sonarr to understand BTNs series-specific ' Overtime' suffix, can you?
So this is largely by design.

**Comment by adamwinn** [comment]
I didn't know E1704 was a valid episode number.  Seems to me like the season was accidentally put into the episode place.

**Comment by Taloth** [comment]
it's just tvdb's special episode number. They chose that particular number ad-hoc, because they have no way to properly mark Overtime as an extra for that particular episode.

**Comment by adamwinn** [comment]
@Taloth was fix 8102cb63ae9e71ab58696573c339f22689a480f5 for this issue?

I'm on Sonarrv3 now and I'm seeing this in the logs
```
19-4-6 21:53:13.4|Info|NzbSearchService|Searching 2 indexers for [Real Time with Bill Maher] Specials
19-4-6 21:53:13.4|Debug|BroadcastheNet|Downloading Feed http://api.broadcasthe.net/: getTorrents("(removed)", {"Category":"Episode","Name":"%Overtime+April+5+2019%","Tvdb":"72231"}, 100, 0)
19-4-6 21:53:13.4|Trace|HttpClient|Req: [POST] http://api.broadcasthe.net/: getTorrents("(removed)", {"Category":"Episode","Name":"%Overtime+April+5+2019%","Tvdb":"72231"}, 100, 0)
19-4-6 21:53:13.4|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
19-4-6 21:53:13.9|Trace|HttpClient|Res: [POST] http://api.broadcasthe.net/: 200.OK (498 ms)
19-4-6 21:53:13.9|Debug|NzbSearchService|Total of 0 reports were found for [Real Time with Bill Maher] Specials from 2 indexers
```
No release group that I can tell releases this series with the date format for the Overtime special
btn:
`Real Time with Bill Maher - S17E11 - Overtime`

hdb:
`Real Time with Bill Maher S17E11 Overtime`

mtv:
`Real.Time.with.Bill.Maher.S17E11.Overtime`

Perhaps a search like `S17E11%Overtime` would be better to do, instead of `%Overtime+April+5+2019%`

You could grab the `S17E11` if the series is defined as Standard and then add on a `%Overtime`.  The string `S17E11%Overtime` matches on BTN

**Comment by Taloth** [comment]
No, that commit was not for this issue. It was for generic specials, such as the xmas special of Doctor Who.
Sonarr only searches by the special episode title, not anything else. Overtime would require special code in Sonarr specifically for that show, which is not something we want to do.
